pkg/cli/doctor.go

```diff
@@ -313,6 +313,19 @@ FROM system.namespace`
 	stmt = `
 SELECT id, status, payload, progress FROM "".crdb_internal.system_jobs
 `
+	checkSystemJobs := `SELECT count(*) FROM "".crdb_internal.system_jobs LIMIT 1`
+	_, err = sqlConn.QueryRow(ctx, checkSystemJobs)
+	// On versions before 23.1, we have a new table called crdb_internal.system_jobs,
```

Comment:
nit: as @rafiss [brought up a good call](https://github.com/cockroachdb/cockroach/pull/104641#discussion_r1228801810) that it should be `after 23.1` instead of `before`.

Reply:
Good point, let me fix that